pkl-formatter/src/main/kotlin/org/pkl/formatter/Builder.kt

```diff
@@ -33,7 +33,7 @@ import org.pkl.parser.syntax.Operator
 import org.pkl.parser.syntax.generic.Node
 import org.pkl.parser.syntax.generic.NodeType
 
-internal class Builder(sourceText: String) {
+internal class Builder(sourceText: String, private val version: CompatVersion) {
```

Comment:
Whatever name we settle on, we should change this and the class name too.

```suggestion
internal class Builder(sourceText: String, private val grammarVersion: GrammarVersion) {
```

Reply:
Changed.